examples/with-static-export/pages/post.js

```diff
@@ -0,0 +1,31 @@
+import { Component } from 'react'
+import Link from 'next/link'
+import Head from 'next/head'
+import fetch from 'isomorphic-fetch'
```

Comment:
shouldn't `fetch` be imported dynamically in `getInitialProps` method so that it's not bundled in the static export?